pkg/util/log/crash_reporting.go

```diff
@@ -87,9 +88,16 @@ func ReportPanic(ctx context.Context, r interface{}, depth int) {
 	// at least somewhat helpful in telling us where crashes are coming from.
 	// We capture the full stacktrace below, so we only need the short file and
 	// line here help uniquely identify the error.
-	file, line, _ := caller.Lookup(depth + 3)
-
-	reportable := fmt.Sprintf("%s %s:%d", format(r), filepath.Base(file), line)
+	// Some exceptions, like a runtime.Error, are assumed to be fine as-is.
```

Comment:
Indeed, the spec in says that the concrete values that implement runtime.Error are unspecified, so there's certainly not a hard commitment as to what they will or will not contain.

That said, the assumption here is based on inspection what is and has been so far defined as `runtime.Errors`: almost all string consts, except a few places where they names of types get included (comparisons and casts).